sumy/summarizers/text_rank.py

```diff
@@ -64,9 +64,11 @@ def _create_matrix(self, document):
         sentences_count = len(sentences_as_words)
         weights = numpy.zeros((sentences_count, sentences_count))
 
-        for i, words_i in enumerate(sentences_as_words):
-            for j, words_j in enumerate(sentences_as_words):
-                weights[i, j] = self._rate_sentences_edge(words_i, words_j)
+        for i in range(0, sentences_count-1):
```

Comment:
I believe you should do `range(0, sentences_count)` here because `list(range(0, 3)) == [0, 1, 2]`. But I would prefer you return `enumerate(sentences_as_words)`. I think the one last missing word may a reason why [the test fails on CI](https://travis-ci.org/miso-belica/sumy/jobs/652782734). So the code after the changes will be:

```python
for i, words_i in enumerate(sentences_as_words):
  for j in range(i+1, sentences_count):
    rating = self._rate_sentences_edge(words_i, sentences_as_words[j])
    weights[i, j] = rating
    weights[j, i] = rating